### PR TITLE
add lib sources for mac needed

### PR DESCRIPTION
**Pull request by drelaptop**
issue, https://github.com/cocos2d/cocos2d-x-external/issues/11

- add all source mac needed
- not include box2d, since added by @crazyhappygame  earlier 
- added libs, other platforms needed NEXT

### CODE REVIEW

File: src/external/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+rm -Rf ../../contrib/tarballs/libbullet-git && git clone --branch master https://github.com/bulletphysics/bullet3 ../../contrib/tarballs/libbullet-git && (cd ../../contrib/tarballs/libbullet-git && git checkout  19f999a) && rm -Rf ../../contrib/tarballs/libbullet-git/.git && (cd ../../contrib/tarballs/ && tar cvJ libbullet-git) > ../../contrib/tarballs/libbullet-git.tar.xz && rm -Rf ../../contrib/tarballs/libbullet-git
+
+rm -Rf ../../contrib/tarballs/libbox2d-git && git clone --branch master https://github.com/erincatto/Box2D ../../contrib/tarballs/libbox2d-git && (cd ../../contrib/tarballs/libbox2d-git && git checkout  f655c603ba9d83f07fc566d38d2654ba35739102) && rm -Rf ../../contrib/tarballs/libbox2d-git/.git && (cd ../../contrib/tarballs/ && tar cvJ libbox2d-git) > ../../contrib/tarballs/libbox2d-git.tar.xz && rm -Rf ../../contrib/tarballs/libbox2d-git
```
Comment (drelaptop) [inline src/external/README.md:40]: this cmd is used to download box2d on `cocos2d-x-3rd-party-libs-src`, how the existed Box2d source comes, which commit? both is same or not @crazyhappygame

Comment (crazyhappygame) [reply src/external/README.md:40]: @drelaptop please replace box2d with proper version

Comment (drelaptop) [reply src/external/README.md:40]: OK, I will new another PR for box2d @crazyhappygame

Comment (drelaptop) [reply src/external/README.md:40]: compare this box2d with that download one, both are same. so no need to change it

**Pull request merged**